### PR TITLE
Bug fix and feature update to Voice Scope App

### PR DESCRIPTION
**Pull request by MarkBrosche**

### CODE REVIEW

File: marketplaceItems/voiceScopeApp/appResources/appData/resources/js/voiceScope_ui.js [hunk 1]
```diff
@@ -21,6 +43,18 @@ function onScriptEventReceived(data){
             button.checked = false;
         }
         break;
+    case "drawButtonStatus":
+        if (data.value) {
```
Comment (zfox23) [inline marketplaceItems/voiceScopeApp/appResources/appData/resources/js/voiceScope_ui.js]: please replace with `drawFrontCheck.checked = data.value;`

File: marketplaceItems/voiceScopeApp/appResources/appData/resources/js/voiceScope_ui.js [hunk 2]
```diff
@@ -2,14 +2,36 @@
...
+    }));
+}
+drawFrontCheck.addEventListener("click", drawButtonClicked);
+function sliderEvent(){
+    EventBridge.emitWebEvent(JSON.stringify({
+        type: "HEIGHT_SLIDER",
+        value: output.innerHTML
+    }));
+}
+// slider.addEventListener("slider", sliderEvent);
```
Comment (zfox23) [inline marketplaceItems/voiceScopeApp/appResources/appData/resources/js/voiceScope_ui.js]: dead code?

File: marketplaceItems/voiceScopeApp/appResources/appData/resources/voiceScope_ui.html [hunk 3]
```diff
@@ -18,10 +18,26 @@ <h1><br>Voice Scope</h1>
...
         </div>
+        <div id="blue">
+            Draw In Front of everything?
+        </div>
+        <div id="toggles">
+            <input type="checkbox" name="checkbox2" id="checkbox2" value="" class="ios-toggle"/>
+            <label for="checkbox2" class="checkbox-label" data-off="false" data-on="true"></label>
+        </div>
+        <div id="blue">
+            Avatar Height Offset:   <span id="demo"></span>m<br>
```
Comment (zfox23) [inline marketplaceItems/voiceScopeApp/appResources/appData/resources/voiceScope_ui.html]: what is `demo` ?

File: marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js [hunk 4]
```diff
@@ -41,7 +41,7 @@
             case "EVENT_BRIDGE_OPEN_MESSAGE":
                 ui.sendToHtml({
                     type: "buttonStatus",
-                    value: enabled
+                    value: Settings.getValue("voiceScopeEnabled")
```
Comment (zfox23) [inline marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js]: please add a default value for this Settings key for when it doesn't exist, i.e.:
`Settings.getValue("voiceScopeEnabled", false);`

File: marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js [hunk 5]
```diff
@@ -51,6 +51,22 @@
                     value: enabled
                 });
                 break;
+            case "TOGGLE_DRAW":
```
Comment (zfox23) [inline marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js]: where were the Toggle Draw checkbox and Height Offset Slider features requested, documented, or scoped? These are additional features, not bugfixes

File: marketplaceItems/voiceScopeApp/appResources/voiceScope.app.json [hunk 6]
```diff
@@ -1,4 +1,4 @@
 {
-    "scriptURL": "./appData/voiceScope_app.js",
-    "homeURL": "./appData/resources/voiceScope_ui.html"
+    "scriptURL": "http://mpassets.highfidelity.com/723aa413-e6a2-4753-b6aa-c9d34390b054-v1/appResources/appData/voiceScope_app.js",
```
Comment (zfox23) [inline marketplaceItems/voiceScopeApp/appResources/voiceScope.app.json]: relative paths please!

File: marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js (hunk 5 above)
Comment (MarkBrosche) [reply marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js]: Captured on the market place trello card, they were some of the immediate feedback I received from users and Philip in face to face conversations.

**Pull request merged**